community/index/src/test/java/org/neo4j/index/internal/gbptree/GBPTreeTest.java

```diff
@@ -436,6 +449,66 @@ public void shouldAllowClosingWriterMultipleTimes() throws Exception
         // THEN that should be OK
     }
 
+    @Test
```

Comment:
I think it's by contract, at least the javadoc says so. I can add a test regardless.